source/main/index.js

```diff
@@ -83,31 +85,17 @@ if (isBlankScreenFixActive) {
 EventEmitter.defaultMaxListeners = 100; // Default: 10
 
 app.allowRendererProcessReuse = true;
-const safeExit = async () => {
-  pauseActiveDownloads();
-  if (!cardanoNode || cardanoNode.state === CardanoNodeStates.STOPPED) {
-    logger.info('Daedalus:safeExit: exiting Daedalus with code 0', { code: 0 });
-    return safeExitWithCode(0);
-  }
-  if (cardanoNode.state === CardanoNodeStates.STOPPING) {
-    logger.info('Daedalus:safeExit: waiting for cardano-node to stop...');
-    cardanoNode.exitOnStop();
-    return;
-  }
-  try {
-    const pid = cardanoNode.pid || 'null';
-    logger.info(`Daedalus:safeExit: stopping cardano-node with PID: ${pid}`, {
-      pid,
-    });
-    await cardanoNode.stop();
-    logger.info('Daedalus:safeExit: exiting Daedalus with code 0', { code: 0 });
-    safeExitWithCode(0);
-  } catch (error) {
-    logger.error('Daedalus:safeExit: cardano-node did not exit correctly', {
-      error,
-    });
-    safeExitWithCode(0);
+
+const getCurrentRtsFlags = () => {
+  const rtsFlagsFromStorage = getRtsFlagsSettings(network);
+  if (!rtsFlagsFromStorage) {
+    if (os.totalmem() < MINIMUM_AMOUNT_OF_RAM_FOR_RTS_FLAGS) {
```

Comment:
I just might not understand the topic and I know you are not the author of this code, but shouldn't this condition be stated other way around?
```suggestion
    if (os.totalmem() >= MINIMUM_AMOUNT_OF_RAM_FOR_RTS_FLAGS) {
```
So if there is at least MINIMUM_AMOUNT_OF_RAM_FOR_RTS_FLAGS of memory then we can use RTS_FLAGS

Reply:
The statement is correct. The name is very misleading. I'll soon push a fix for it